tests/integration/defs/perf/pytorch_model_config.py

```diff
@@ -90,6 +90,25 @@ def get_model_yaml_config(model_label: str,
                 'enable_attention_dp': False,
             }
         },
+        # Qwen3 models with fp4 quantization
+        {
+            'patterns': [
+                'qwen3_235b_a22b_fp4-bench-pytorch-float4-maxbs:512-maxnt:2048-input_output_len:1000,2000-con:512-ep:4-gpus:4',
+                'qwen3_235b_a22b_fp8-bench-pytorch-float8-maxbs:512-maxnt:2048-input_output_len:1000,2000-con:256-ep:8-gpus:8]'
```

Comment:
Where do we add `qwen3_235b_a22b_fp8-bench-pytorch-float8-maxbs:512-maxnt:2048-input_output_len:1000,2000-con:256-ep:8-gpus:8]`?

Also, the comment mention "Qwen3 models with fp4 quantization" but it contains some tests of FP8 and might be confused.